### PR TITLE
Caption mis-positioned with Boostrap4 #100

### LINKED ISSUE #100: Caption mis-positioned with Boostrap4
ss

```scss
/* TEMP FIX */
.CalendarMonth {
  table {
    caption: {
      caption-side: top;
    }
  }
}
.CalendarMonth__caption {
  caption-side: top;
}
```

### PR DESCRIPTION
**Pull request by zeroasterisk**
fixes #100  worked for me locally.

problems occured when combined with bootstrap4:
https://github.com/twbs/bootstrap/blob/a27a488002aafb6725e80f63d7022f9af32df669/dist/css/bootstrap-reboot.css#L386

### CODE REVIEW

File: css/CalendarMonth.scss [hunk 1]
```diff
@@ -14,6 +14,9 @@
   table {
     border-collapse: collapse;
     border-spacing: 0;
+    caption: {
+      caption-side: top;
```
Comment (ljharb) [inline css/CalendarMonth.scss]: should these use `initial` to override bootstrap, rather than specifying "top"?

Comment (zeroasterisk) [inline css/CalendarMonth.scss]: maybe... but wouldn't anything other than `top` break your styling?

Comment (ljharb) [inline css/CalendarMonth.scss]: Since we don't currently specify a caption-top, I'd think we want the browser default.

Comment (bsr203) [inline css/CalendarMonth.scss]: `initial` also worked for me. thanks

**Comment by majapw** [comment]
Hmm, I think I can't push to your master branch I'll reopen these changes with comments addressed in a separate PR.

**Pull request closed without merging**